### PR TITLE
Improve formatting and add more definitions to EIP-634

### PR DESCRIPTION
**Pull request by axic**
This is the draft I've mentioned in https://github.com/ethereum/EIPs/issues/2439#issuecomment-614833124

### CODE REVIEW

**Comment by axic** [comment]
@ricmoo I had this locally forever by now, hope it is useful.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 634 requires approval from one of (@ricmoo)

**Comment by ricmoo** [comment]
These changes look good except for the new `vnd*` keys. I will be updating this week with the new reverse-dot notation discussed on the discussion-to link. :)

**Comment by axic** [comment]
> These changes look good except for the new vnd* keys.

What is the problem with the keys?

There are 3 outstanding changes to 634: #2614, #2623, and this. Perhaps it would be good just to merge them, and then you can do another sweep/rewrite whenever you get around to do it? :wink:

File: EIPS/eip-634.md [hunk 1]
```diff
@@ -45,13 +46,15 @@ services must be prefixed with `vnd.`.
...
 
 - **vnd.github** - a GitHub username
-- **vnd.peepeth** - a peepeth username
-- **vnd.twitter** - a twitter username
-
+- **vnd.gitlab** - a GitLab username
+- **vnd.peepeth** - a Peepeth username
+- **vnd.twitter** - a Twitter username
+- **vnd.keybase** - a Keybase username
+- **vnd.reddit** - a Reddit username
```
Comment (axic) [inline EIPS/eip-634.md:58]:
```suggestion
- **vnd.telegram** - a Telegram username

```

To address part of https://github.com/ethereum/EIPs/issues/2439#issuecomment-614827905

**Pull request closed without merging**